sem/HW/hw_6/goods.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel, Field
+
+
+class GoodsIn(BaseModel):
+    name: str = Field(max_length=32)
+    description: str = Field(max_length=500)
+    price: float
+
+
+class Goods(BaseModel):
+    id: int
+    name: str = Field(max_length=32)
+    description: str = Field(max_length=500)
+    price: float
```

Comment:
Заметьте, что модели практически идентичны, просто у одной и есть дополнительное поле. Почему тогда не наследоваться?
```suggestion
class Goods(GoodsIn):
    id: int
```